lib/proxy.js

```diff
@@ -698,9 +740,9 @@ function redirectToHttps(req, res, target, ssl, log) {
   req.url = req._url || req.url; // Get the original url since we are going to redirect.
 
   var targetPort = ssl.redirectPort || ssl.port;
-  var hostname = req.headers.host.split(':')[0] + ( targetPort ? ':' + targetPort : '' );
-  var url = 'https://' + path.join(hostname, req.url);
-  log && log.info('Redirecting %s to %s', path.join(req.headers.host, req.url), url);
+  var hostname = req.headers.host.split(':')[0] + (targetPort ? ':' + targetPort : '');
+  var url = 'https://' + path.posix.join(hostname, req.url);
```

Comment:
why is better to use posix in this case?

Reply:
On windows, `path.join` would use `\`.